### ISSUE TITLE
setting collateral on multiple machines

### ISSUE THREAD

**Issue by Grondthehammer** [Berry-Pool/nami-wallet#191]
I'm not very technical when it comes to this. This is my interpretation of the issue I am facing, I'd appreciate any input!

I've created a Nami wallet that I've been using on my desktop but have recently restored it to my laptop so I can use it remotely. It seems like every time I want to delist/alter a smart contract NFT listing I need to reset the collateral for the wallet on the computer I'm currently using. I am on my laptop now and I'm not able to delist an NFT on cnft.io because after setting the collateral on my laptop it claims the collateral has been removed from the asset ("posted collateral is invalid or spent") when I click 'unlist'. I assume this is because I posted that SC listing on my desktop and set the collateral there, and this collateral gets unset when I set it on the laptop? Is there a way this can be resolved so I can harmoniously use the same Nami wallet on two or more devices? Thank you :)

**Comment by alessandrokonrad** [comment]
Hey, yeah that is kinda tricky, because Nami only seperates the collateral amount locally, so the other instance doesn't know about it. The collateral thing is also something that can be removed in the future, it's just temporary because of some security reasons, but not necessary anymore after some more Cardano updates. For now we have to live with it :)

**Comment by QCPOLstakepool** [comment]
I'm also having issues with this. 

1. I set collateral on computer 1
2. I set collateral on computer 2
3. Can't submit TX on SundaeSwap anymore:
    ```

    Jan 23 21:09:44 cardano-mainnet-s01-r02 bash[3101906]: [cardano-tx-submit:Info:36] [2022-01-24 02:09:44.04 UTC] txSubmitPost: failed to submit transaction: transaction submit error ShelleyTxValidationError ShelleyBasedEraAlonzo (ApplyTxError [UtxowFailure (WrappedShelleyEraFailure (UtxoFailure (TooManyCollateralInputs 3 6)))])
    ``` 

It seems that Nami consider **all** UTXO of 5000000 lovelace to be a collateral when it should only consider one.

**Comment by alessandrokonrad** [comment]
> I'm also having issues with this.
> 
> 1. I set collateral on computer 1
> 2. I set collateral on computer 2
> 3. Can't submit TX on SundaeSwap anymore:
>    ```
>    
>    Jan 23 21:09:44 cardano-mainnet-s01-r02 bash[3101906]: [cardano-tx-submit:Info:36] [2022-01-24 02:09:44.04 UTC] txSubmitPost: failed to submit transaction: transaction submit error ShelleyTxValidationError ShelleyBasedEraAlonzo (ApplyTxError [UtxowFailure (WrappedShelleyEraFailure (UtxoFailure (TooManyCollateralInputs 3 6)))])
>    ```
> 
> It seems that Nami consider **all** UTXO of 5000000 lovelace to be a collateral when it should only consider one.

Yes I made a little update, where I also allow now utxos between 5-50 ada as collateral. Are you on version 3.1.2? But this looks like Sundaeswap is building the tx wrongs, they include too many utxos in the collateral field

**Comment by Bookdat** [comment]
> Hey, yeah that is kinda tricky, because Nami only seperates the collateral amount locally, so the other instance doesn't know about it. The collateral thing is also something that can be removed in the future, it's just temporary because of some security reasons, but not necessary anymore after some more Cardano updates. For now we have to live with it :)

Any idea when an update will fix this. it is quite annoying when switching between my laptop and destop. 

Thanks.
